fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/services/functions/AnyTypesPredicate.java

```diff
@@ -55,9 +52,8 @@ public AnyTypesPredicate(final String...types) {
     public boolean testThrows(final Node input) throws RepositoryException {
         requireNonNull(input, "null node has no types!");
         if (isFrozen.test(input) && input.hasProperty(FROZEN_MIXIN_TYPES)) {
-            final Property frozenTypesProperty = input.getProperty(FROZEN_MIXIN_TYPES);
-            final List<Value> values = copyOf(property2values.apply(frozenTypesProperty));
-            if (values.stream().map(uncheck(Value::getString)).anyMatch(nodeTypes::contains)) {
+            if (property2values.apply(input.getProperty(FROZEN_MIXIN_TYPES)).map(uncheck(Value::getString))
```

Comment:
So this a style thing, but I would do something like:
```
final Stream<Value> values = property2values.apply(input.getProperty(FROZEN_MIXIN_TYPES));
return values.map(uncheck(Value::getString)).anyMatch(nodeTypes::contains))
|| nodeTypes.stream().anyMatch(uncheck(input::isNodeType));
```
but maybe that's less readable.